app/controllers/inventory_controller.rb

```
@@ -22,4 +28,45 @@ def revise
+    dummy_history_ending = StockHistory.where(stock_id: @stock.id, created_at: end_date..Date.tomorrow)
+    dummy_history_ending.each do |dummy|
+      ending_stock -=dummy.alteration
+    end
+
+    history_track = StockHistory.where(stock_id: @stock.id, created_at: start_date..end_date)
+
+    respond_to do |format|
+      #format.json { render json: {starting_stock: starting_stock, ending_stock: ending_stock} }
+      format.json { render json: {start_date: dummy_start_date, product_track: history_track,   current_stock: current_stock, starting_stock: starting_stock, ending_stock: ending_stock} }
+    end
+  end
```

RuboCop (1.76.1)

[convention] Assignment Branch Condition size for `product_history` is too high. [<27, 44, 2> 51.66/23]

(Metrics/AbcSize)